tools/test-proxy/Azure.Sdk.Tools.TestProxy/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,7 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
```

Comment:
Where did this come from? I'm a bit weirded out it wasn't already there, and I don't understand why it needs to be if it wasn't before.

We don't need this if you swapped that property to `public` right?

Reply:
I made the Logger property internal again so we do need this.